_commontech/Customer_3.0.0_Release_Notes.md

```diff
@@ -92,11 +92,11 @@ You can select from the following compute node extensions to add to your IBM Blu
 
  * You can select Block Storage: SSD or HYBRID
 
-<a name="hipaa"></a>**HIPAA-enabled Cloud:** A HIPAA-enabled cloud has the same configuration and expansion options as any other IBM Blue Box Cloud offering, but we are enabling _over 50 additional information-security controls_ that support our customers who must store personal health information (PHI). Because of this increased information security capability, a HIPAA-enabled cloud must be HIPAA-enabled from its inception: you cannot upgrade an existing cloud to enable HIPAA protocol. Also, you cannot downgrade from an IBM Blue Box Local HIPAA cloud: the existing HIPAA cloud must be wiped—per HIPAA protocol—and you must order a new Local cloud with non-HIPAA controllers.
+<a name="hipaa"></a>**HIPAA-enabled Cloud:** A HIPAA-enabled cloud has the same configuration and expansion options as any other IBM Bluemix on OpenStack offering, but we are enabling _over 50 additional information-security controls_ that support our customers who must store personal health information (PHI). Because of this increased information security capability, a HIPAA-enabled cloud must be HIPAA-enabled from its inception: you cannot upgrade an existing cloud to enable HIPAA protocol. Also, you cannot downgrade from an IBM Bluemix Local on OpenStack HIPAA cloud: the existing HIPAA cloud must be wiped—per HIPAA protocol—and you must order a new Local cloud with non-HIPAA controllers.
 
- * You can now select HIPAA Enablement for your IBM Blue Box Local Cloud.
+ * You can now select HIPAA Enablement for your IBM Bluemix Local on OpenStack.
 
-<a name="pure-app"></a>**Support for Blue Box on PureApp (Beta release):** PureApp is an existing IBM system for hardware management. With its Beta release, we are making it available as a platform for IBM Blue Box Local, in addition to the Cisco hardware already available.
+<a name="pure-app"></a>**Support for Bluemix on OpenStack on PureApp (Beta release):** PureApp is an existing IBM system for hardware management. With its Beta release, we are making it available as a platform for IBM Bluemix Local on OpenStack, in addition to the Cisco hardware already available.
```

Comment:
please refer to PR #421 review comments for information about PureApp and how to change this wording.